controller/proxy-injector/fake/data/pod.patch.json

```diff
@@ -1,353 +1,353 @@
 [
-  {
```

Comment:
What is up with the white-space change?

Reply:
This must be my editor settings (VS Code)